scripts/buildArticles.js

```diff
@@ -88,12 +99,19 @@ getDirectories(articlesDirectory, (err, res) => {
       item["items"] = null;
     }
     item["subcategory"] =
-      filePath.split("/").length > 1 && !filePath.split("/")[1].endsWith(".mdx")
+      filePath.split("/").length > 1 &&
+      (!filePath.split("/")[1].endsWith(".mdx") ||
+        !filePath.split("/")[1].endsWith(".js"))
         ? filePath.split("/")[1]
         : null;
-    item["title"] = data["title"];
-    item["url"] = "/" + filePath.replace("index.mdx", "");
-    item["weight"] = data["weight"];
+    item["title"] = redocTitles[item["subcategory"]]
```

Comment:
It looks very unstable. For instance what if someone puts redoc document as the first file in the category? In that case, it doesn't seem to work.